Contents/MacOS/startupRAMDiskandCacheMover.sh

```diff
@@ -201,6 +201,54 @@ move_chrome_chanary_cache()
    fi
 }
 
+#
+# Brave Cache
+#
+
+# Create a directory for Brave in the ramdisk
+make_brave_dir()
+{
+    /bin/mkdir -pv "${USERRAMDISK}"/Brave-Browser
+}
+
+# Move existing Brave cache to ramdisk
+move_brave_cache()
+{
+    /bin/mkdir -p /tmp/Brave-Browser
+    /bin/mv ~/Library/Caches/BraveSoftware/Brave-Browser/* /tmp/Brave-Browser
+    /bin/mv /tmp/Brave-Browser/* "${USERRAMDISK}"/Brave-Browser
+    /bin/rm -rf ~/Library/Caches/BraveSoftware/Brave-Browser
+    
+}
+
+# Link Brave cache to ramdisk
+link_brave_dir()
+{
+    /bin/ln -v -s "${USERRAMDISK}"/Brave-Browser ~/Library/Caches/BraveSoftware/Brave-Browser
+}
+
+# Check what to do with brave
+check_brave_cache()
+{
+    if [ -d "/Users/${USER}/Library/Caches/BraveSoftware/Brave-Browser" ]; then
+        if user_response "${MSG_PROMPT_FOUND}" 'Brave'"${MSG_MOVE_CACHE}" ; then
+            close_app "Brave Browser"
+            make_brave_dir
+            move_brave_cache
+            link_brave_dir
```

Comment:
consider replacing this line with line 225:
```
     /bin/ln -v -s "${USERRAMDISK}"/Brave-Browser ~/Library/Caches/BraveSoftware/Brave-Browser 
```
it is used once only, does not needs to be a func